public/client.js

```diff
@@ -0,0 +1,133 @@
+var ViewControlsDisplay =  Backbone.View.extend({/*properties*/});
+var viewControlsDisplay = new ViewControlsDisplay({el: '#controlsdisplay'});
+
+var ViewListDisplay =  Backbone.View.extend({/*properties*/});
+var viewListDisplay = new ViewListDisplay({el: '#catalogue'});
+
+var ViewAddDisplay = Backbone.View.extend({/*properties*/});
+var viewAddDisplay = new ViewAddDisplay({el: '#adddisplay'});
+
+var rcRecord = new Backbone.Model.extend({
+  initialize: function(){console.log("rcRecord initialized")}
+              });
+var newRcRecord = new RcRecord();
```

Comment:
I changed that and now am getting a backbone error.
I’ve committed the changes

From: jamdon 
Sent: Friday, June 17, 2016 2:34 PM
To: JSCRIPT300-Spring2016/Final_Project 
Cc: seancava ; Author 
Subject: Re: [JSCRIPT300-Spring2016/Final_Project] Final Project Proposal (#10)

In public/client.js:

> @@ -0,0 +1,133 @@
> +var ViewControlsDisplay =  Backbone.View.extend({/*properties*/});
> +var viewControlsDisplay = new ViewControlsDisplay({el: '#controlsdisplay'});
> +
> +var ViewListDisplay =  Backbone.View.extend({/*properties*/});
> +var viewListDisplay = new ViewListDisplay({el: '#catalogue'});
> +
> +var ViewAddDisplay = Backbone.View.extend({/*properties*/});
> +var viewAddDisplay = new ViewAddDisplay({el: '#adddisplay'});
> +
> +var rcRecord = new Backbone.Model.extend({
> +  initialize: function(){console.log("rcRecord initialized")}
> +              });
> +var newRcRecord = new RcRecord();
You defined your model constructor as rcRecord (line 10) but you're calling it RcRecord here.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.